istio/src/main/scala/io/buoyant/k8s/istio/IstioResponse.scala

```diff
@@ -0,0 +1,11 @@
+package io.buoyant.k8s.istio
+
+import com.twitter.util.Duration
+
+case class IstioResponse(statusCode: Int, duration: Duration) extends IstioDataFlow {
```

Comment:
Any particular reason why IstioRequest is parameterized on the underlying request type and contains the underlying request but IstioResponse is not parameterized and does not include the underlying response?

Reply:
Fixed on the same as above